### PR TITLE
Forbole

### PR DESCRIPTION
**Pull request by Naviol**

### CODE REVIEW

File: standard_crypto/server_ca.crt [hunk 1]
```diff
@@ -1,13 +0,0 @@
------BEGIN CERTIFICATE-----
```
Comment (EricBolten) [inline standard_crypto/server_ca.crt:1]: You'll have to undo your deletion of Standard Crypto's cert.

File: stewards.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [
     {
-        "ca": "standard_crypto/server_ca.crt",
-        "endpoint": "https://steward.sommelier.standardcryptovc.com:5734"
+        "ca": "forbole/server_ca.crt",
+        "endpoint": "https://steward.sommelier.forbole.com:5734"
```
Comment (EricBolten) [inline stewards.json:4]: You need to append an entry to this list rather than overwriting it. You'll need to merge main and update.

**Review by EricBolten — CHANGES_REQUESTED** [review]
Some changes needed.

File: stewards.json (hunk 2 above)
Comment (Naviol) [reply stewards.json:20]: Done

**Pull request closed without merging**